boot/zephyr/include/target.h

```diff
@@ -33,8 +33,7 @@
 /*
  * Sanity check the target support.
  */
-#if !defined(DT_FLASH_DEV_NAME) || \
```

Comment:
Instead pleas add sanity check for case:
!defined(CONFIG_XTENSA) && defined(DT_FLASH_DEV_NAME)
defined(CONFIG_XTENSA) && defined(DT_SPI_NOR_DRV_NAME)

Reply:
Done.